packages/atoms/src/Hint.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Text as RNText, View } from 'react-native';
+import { useNativeStyles, prepareNativeStyle } from '@trezor/styles';
+import { Icon } from './Icon/Icon';
+import { IconType } from './Icon/iconTypes';
+import { Color, colorVariants } from '@trezor/theme';
+import * as CSS from 'csstype';
+
+type HintType = 'hint' | 'error';
+
+type HintProps = {
+    type: HintType;
+    children?: React.ReactNode;
+};
+
+const hintStyle = prepareNativeStyle(() => ({
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+}));
+
+const hintTypeStyle = prepareNativeStyle<{ color: CSS.Property.Color }>((utils, { color }) => ({
+    ...utils.typography.label,
+    color,
+    marginLeft: 6,
+}));
+
+export const Hint = ({ type, children }: HintProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    const hintTypeProps: Record<
+        HintType,
+        { iconType: IconType; color: CSS.Property.Color; colorVariant: Color }
+    > = {
+        hint: {
+            colorVariant: 'gray600',
```

Comment:
I think you can make this much more simple. You don't need color defined twice:

```tsx
// this could be placed outside component because color is just string now
const hintVariants: Record<
         HintType,
         { iconType: IconType; color: Color }
     > = {
         hint: {
             colorVariant: 'gray600',
	     iconType: 'question',
	},
	...
}
```

```tsx
...
<Icon
                     type={hintVariants[variant].iconType}
                     color={hintVariants[variant].color}
                     size="tiny"
                 />
                 <Text style={applyStyle(hintTypeStyle, { color: hintVariants[variant].color })}>
...
```

```tsx
const hintTypeStyle = prepareNativeStyle<{ color: Color }>((utils, { color }) => ({
     ...utils.typography.label,
     color: utils.colors[color],
     marginLeft: 6,
 }));
```